client/ayon_deadline/plugins/publish/global/collect_jobinfo.py

```diff
@@ -293,13 +293,13 @@ def _get_artist_overrides(cls, overrides, profile):
             EnumDef(
                 "primary_pool",
                 label="Primary pool",
-                default="none",
+                default=default_values.get("primary_pool"),
```

Comment:
Shouldn't both of these still fall back to what the values were originally, which is the string `"none"` it seems?
```suggestion
                default=default_values.get("primary_pool", "none"),
```

Reply:
Yeah good point. Lemme test how this behaves if I don't set any overrides for pools in the profile as i believe this is what "none" refers to here. 
Or the actual none pool in deadline. I'll check ✌️

Reply:
The `None` would just fallback to the "first entry". I think explicitly stating "none" may be clearer or more explicit?